### ISSUE TITLE
Import error with 1.98.5 save

### ISSUE THREAD

**Issue by A-Master-of-Puppets** [Innoxia/liliths-throne-public#123]
When I try to import a save from 1.98.5, I get an error ([log](https://gist.github.com/A-Master-of-Puppets/0f9c0c392a288ff3fdfe63f0de062600)).
This appears to be because the location SHOPPING_ARCADE_ASHLEYS_SHOP doesn't exist in the save file, so calling getCell on it returns null. However, that location _also_ doesn't exist in saves from 1.99, but those load just fine. I don't understand the game code well enough to help with this one.

**Comment by A-Master-of-Puppets** [comment]
This is the weirdest thing, but I went to have another go at fixing this so I could play with the characters affected, and I can't reproduce it. I'm loading the exact same save files, and they work fine. I've rebuilt the game a few times working on other things, but I certainly haven't touched anything in the guts of the engine, and I'd _swear_ that I was seeing the error with 0.2.0.2, yet that's all I can think of; I somehow managed to mix up 0.2.0 with 0.2.0.2 and the hotfix fixed it. All that's left is that the potions from old saves fail to import, so I've whipped up a quick fix for that if you're interested. #155
